pkg/cli/build_test.go

```diff
@@ -0,0 +1,28 @@
+package cli
+
+import (
+	"testing"
+
+	"github.com/containers/buildah/define"
+)
+
+func TestGetFormat(t *testing.T) {
+	_, err := GetFormat("bogus")
+	if err == nil {
+		t.Error("GetFormat should have failed.")
+	}
```

Comment:
Using `github.com/stretchr/testify/assert.NotNil()` and friends in here would shorten this up.